src/sdk/sql_router.h

```diff
@@ -76,6 +76,8 @@ class SQLRouter {
 
     virtual bool DropDB(const std::string& db, hybridse::sdk::Status* status) = 0;
 
+    virtual bool DropTable(const std::string& db, const std::string& table, hybridse::sdk::Status* status) = 0;
```

Comment:
why add `DropTable` interface? We can execute drop cmd with `ExecuteDDL`

Reply:
I will remove this if we have supported `drop table` in SQL.